integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/DpvConversationStrategy.java

```diff
@@ -3,13 +3,16 @@
 import no.altinn.services.serviceengine.correspondence._2009._10.InsertCorrespondenceV2;
 import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
 import no.difi.meldingsutveksling.nextmove.logging.ConversationResourceMarkers;
+import no.difi.meldingsutveksling.noarkexchange.NoarkClient;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'no.difi.meldingsutveksling.noarkexchange.NoarkClient'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AUselessImportCheck)